### DISCUSSION TITLE
Exclude pages from certain applications

### DISCUSSION THREAD

**Discussion by pimjansen** [tarampampam/error-pages#133] [General]
Hey,

I have added the error pages to my k8s nginx-ingress. It works fine and proper error pages are returned. The problem i have is that i have some applications that handle 500 / 404 etc theirselfs with their own custom error pages. At this moment all my 404 responses are catched and return the error pages from the defaultHandler.

Is there a proper way to configure that certain applications can do their own?

**Comment by tarampampam** [comment]
Hi @pimjansen! I have no idea because I haven't much experience with the k8s. But maybe @onedr0p can help you?

**Reply by pimjansen** [reply]
I see that i can do something native on the nginx ingress itself. Will let you know

**Comment by onedr0p** [comment]
@pimjansen instead of having a global setting for the error codes you would need to set it on a per-ingress basis. See the ingress nginx annotations docs on that.

**Reply by tarampampam** [reply]
Wow! What a reaction speed! 🔥

**Reply by pimjansen** [reply]
> @pimjansen instead of having a global setting for the error codes you would need to set it on a per-ingress basis. See the ingress nginx annotations docs on that.

Yeah i figured but it cant go the other way around it seems. So that it behaves as fallback. I now have a 501/504 as global in case of pod disruption and a custom 404 for the application that does not handle it properly itself.
